tools/nni_trial_tool/log_utils.py

```diff
@@ -134,7 +134,7 @@ def __init__(self, logger, log_collection, log_level=logging.INFO):
         self._is_read_completed = False
         self.process_exit = False
         self.log_collection = log_collection
-        self.log_pattern = re.compile(r'^NNISDK_MEb\'.*\'$')
+        self.log_pattern = re.compile(r'NNISDK_MEb\'.*\'$')
```

Comment:
I suggest to keep '^', it makes the matching more restrict.

Reply:
The issue is caused by ^, since that case doesn't have \r\n